doc/packaging.md

```diff
@@ -315,9 +315,9 @@ Notes on behavior:
 
 - The imported device CA will now be referred to as the "Edge CA".
 
-- The master identity key and master encryption key are two symmetric keys dynamically generated by `iotedged` for internal use. They will not be imported into the new services.
+- The workload API exposes an encrypt and decrypt API for modules which uses module-specific keys derived from the module ID and a master encryption key. The master encryption key will be imported into the new services via an `imported_master_encryption_key` property.
```

Comment:
We don't want to document the property. It's a hack that only exists for `import`'s sake.

Reply:
Removed